cmd/clusterctl/client/alpha/rollout_resumer.go

```diff
@@ -53,3 +67,11 @@ func resumeMachineDeployment(proxy cluster.Proxy, name, namespace string) error
 
 	return patchMachineDeployment(proxy, name, namespace, patch)
 }
+
+// resumeKubeadmControlPlane removes paused annotation.
+func resumeKubeadmControlPlane(proxy cluster.Proxy, name, namespace string) error {
+	pausedAnnotation := strings.Replace(clusterv1.PausedAnnotation, "/", "~1", -1)
```

Comment:
Can we have a comment / link to why we need to replace that? E.g. to https://datatracker.ietf.org/doc/html/rfc6901#section-3